models/adlink.go

```diff
@@ -0,0 +1,68 @@
+package models
+
+import (
+	"github.com/beego/beego/v2/client/orm"
+)
+
+type AdLink struct {
+	Base
+
+	Result          *Result  `orm:"rel(fk)"`
+	Type            string   `orm:"type(text)"`
+	Position        string   `orm:"type(text)"`
+	Link            string   `orm:"type(text)"`
+}
+
+const (
```

Comment:
Where we are using it? it seems only using from the test 🤔

Reply:
@junan This will be used in [another PR](https://github.com/carryall/go-google-scraper-challenge/pull/40) to fill the `Adlink`'s position and type fields 😄